app/src/views/animations/status-animations.js

```diff
@@ -47,10 +48,38 @@ export default class StatusAnimations {
       }
       direction *= -1;
     }
+    frames.push(() => [[lastPanel, 0, "error"]]);
 
     StatusAnimations.animateStrips(view, ['0', '1', '2'], frames, 50, completeCallback);
   }
 
+  // FIXME: This is a hack to support failure animation on one panel
```

Comment:
I think this would be good to tackle when we next overhaul the animation system.